lib/platform/gitlab/index.ts

```diff
@@ -204,31 +206,33 @@ export async function initRepo({
   return repoConfig;
 }
 
-export function getRepoForceRebase() {
+export function getRepoForceRebase(): bool {
   return false;
 }
 
-export async function setBaseBranch(branchName = config.baseBranch) {
+export async function setBaseBranch(
+  branchName = config.baseBranch
```

Comment:
It looks like codecov is complaining about this line not being covered now: https://codecov.io/gh/renovatebot/renovate/compare/8d0aa39920bf004efa957a09cb8a9bd1ee624f23...c08578043f95c7ded016172cd3b33b0e22a7ff0f/diff#D2-214

Reply:
It is also that line: https://codecov.io/gh/renovatebot/renovate/compare/8d0aa39920bf004efa957a09cb8a9bd1ee624f23...4b056f3c04869ff6ef2ad16b8ef78277c37fa69c/src/lib/platform/gitlab/index.ts?before=lib/platform/gitlab/index.ts#L520

Reply:
not sure what is the best way to test `await gitlab.deleteBranch('branch', false);` (I guess codecov is complaining because we are not testing with the default)